kernel/packages/unity-interface/UnityInterface.ts

```diff
@@ -513,6 +513,11 @@ export class UnityInterface {
   }
 
   private SendMessageToUnity(object: string, method: string, payload: any = undefined) {
+    if (!this.Module) {
```

Comment:
Could you be more verbosic about this? or probably adding a comment like `"this.Module" is not present in using remote websocket renderer`?

It is very hard to understand that if without reading the commit message